### ISSUE TITLE
External Pixel Weight as Input()

### ISSUE THREAD

**Issue by ncalik** [vlfeat/autonn#26]
Hi,
I'm trying to use autoNN for segmentation. When I use pixel weight, I've got an error.
How can I add weights as an input? Thanks...
-------------------------------------------------------------------------------------------------------
Loss Func: 
Weigts_ = Input('W');
loss =   sum( sum(  Weigts_.*(imgs5 - label).^2  , 2) ,1)  ./...
        (size(imgs5,1)*size(imgs5,2));
-------------------------------------------------------------------------------------------------------
Error : 
Struct contents reference from a non-struct array object.
Error in Net/eval (line 69)
    if numel(net.inputVarsInfo) < var || ~isequal(net.inputVarsInfo{var}.size,size(net.vars{var}))
Error in seg (line 125)
      net.eval({'images', images, 'labels', labels,'W',W_mat }) ;
-------------------------------------------------------------------------------------------------------

**Comment by ryanwebster90** [comment]
I believe this issue was resolved here #25 . Try updating and re running those lines.

**Comment by ncalik** [comment]
It works well after updating codes..
Thanks for help..
